system/jlib/jflz.cpp

```diff
@@ -1001,14 +1001,13 @@ IExpander *createFastLZExpander()
 
 #define FLZ_BUFFER_SIZE (0x100000)
 
-#define FLZCOMPRESSEDFILEFLAG (I64C(0xc3518de42f15da57))
-
+static const __uint64 FLZCOMPRESSEDFILEFLAG = U64C(0xc3518de42f15da57);
 
 struct FlzCompressedFileTrailer
 {
     offset_t        zfill1;             // must be first
     offset_t        expandedSize;
-    __int64         compressedType;
+    __uint64        compressedType;
```

Comment:
Might be better to leave this as was and change the #define to a static const __int64 ... as I did elsewhere in similar circumstances.